### DISCUSSION TITLE
Ограничение по нескольким странам

### DISCUSSION THREAD

**Discussion by artzavod** [cdek-it/widget#60] [Ideas]
Добрый день! Судя по документации можно либо разрешить все страны (country=all), либо задать ОДНУ конкретную. А как можно разрешить выбирать ПВЗ в "России,Беларуссии,Казахстане"?

**Comment by vermorag** [comment]
Добрый день. Такого функционала в виджете нет. Можете подсказать какой бизнес процесс его использования?

**Reply by artzavod** [reply]
Привествую! Ну мы интренет магазин, производим товар и доставляем в перечисленные страны. Клиенты при оформлении заказа, сами выбирают транспортную компанию и ПВЗ. Сейчас в настройках виджета выставлено ограничение country='Россия', но хотелось бы добавить ещё Беларуссию и Казахстан. А сейчас если сделать country=all, то вылезает слишком много лишних стран/городов.

**Reply by vermorag** [reply]
На странице чекаута используется только виджет и никаких дополнительных полей для адреса/страны/города?

**Reply by artzavod** [reply]
Если при заказе выбрали доставку в ПВЗ и выбрали в списке сдек, то больше никаких полей, нужно только выбрать нужный ПВЗ на карте.

**Reply by vermorag** [reply]
Понял. Завел задачу на доработку. Номер задачи - CMS-880. 
Если требуется, можете подключиться к рассылке по релизу всех новых версий наших плагинов. Письма в рассылке о новых версиях содержат номера задач команды разработки, входящих в патч, а также краткое описание обновлений.
Для подписки на рассылку напишите, пожалуйста, в телеграм: https://t.me/cdek_it_bot , раздел Интеграция для интернет-магазинов.

**Reply by artzavod** [reply]
Добрый день! Есть новости по этой теме?
